overlay.py

```diff
@@ -52,9 +52,7 @@ def __init__(self, bike, width=1280, height=740, bg: str = None):
         self.client.on_log = self.on_log
 
         # Set the camera status to offline if connection breaks
-        is_online_topic = (
-            f"{str(DAShboard.video_feed_status_root)}/{self.device}"
-        )
+        is_online_topic = str(DAShboard.status_video_feed)
```

Comment:
We still need to specify the device here!

![image](https://user-images.githubusercontent.com/17876556/91370827-b3d0f680-e852-11ea-8931-dbfbd4c654a5.png)

Reply:
Interesting thing to note that we might want to change - The camera publishes `"online": true` immediately before disconnecting on `OpenCVStaticImageBackend`, because `_is_video_on` always returns `True`. Should probably change that to check if the window has been closed!

(Even with that fixed we'd still get `"online": false` twice in a row, but at least that's more accurate)

Reply:
Fixed the will message topic to be for each device (it was the classic "did a manual refactor and forgot to change something") and hardcoded the False value for the exit message.
1886a0d